client/src/lib/components/SearchInput/SearchInput.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import type { SelectOption } from "src/definitions/form";
+
+  import { createEventDispatcher } from "svelte";
+
+  const dispatch = createEventDispatcher<{ search: SelectOption }>();
+
+  export let name: string;
+  export let id = `${name}-label`;
+  export let options: Array<SelectOption> = [];
+  export let placeholder = "";
+
+  let searchInput: HTMLInputElement;
+
+  const listName = `${name}-list`;
+
+  let selectedOption: SelectOption | undefined;
+
+  const handleInput = (e: any) => {
+    const foundOption = options.find((item) => item.label === e.target.value);
+
+    if (foundOption) {
+      selectedOption = foundOption;
+    }
+  };
+
+  const handleKeypress = (e: KeyboardEvent) => {
+    if (e.key === "Enter") {
+      e.preventDefault();
+    }
+
+    if (selectedOption && e.key === "Enter") {
+      dispatch("search", selectedOption);
+      searchInput.value = "";
+      selectedOption = undefined;
+    }
+  };
+</script>
+
+<div class="fr-select-group">
```

Comment:
(Nit) Ça n'a pas l'air d'avoir d'effet visuel mais ça semble préférable d'être cohérent ici

```suggestion
<div class="fr-input-group">
```